src/uct/ib/rc/base/rc_ep.c

```diff
@@ -79,12 +79,17 @@ void uct_rc_txqp_vfs_populate(uct_rc_txqp_t *txqp, void *parent_obj)
                             &txqp->available, UCS_VFS_TYPE_I16, "available");
 }
 
-ucs_status_t uct_rc_fc_init(uct_rc_fc_t *fc, int16_t winsize
+ucs_status_t uct_rc_fc_init(uct_rc_fc_t *fc, uct_rc_iface_t *iface
                             UCS_STATS_ARG(ucs_stats_node_t* stats_parent))
 {
     ucs_status_t status;
 
-    fc->fc_wnd = winsize;
+    fc->fc_wnd = iface->config.fc_wnd_size;
+
+    if (!iface->config.fc_enabled) {
+        UCS_STATS_NODE_ASSIGN(&fc->stats, NULL);
```

Comment:
UCS_STATS_NODE_RESET(&fc->stats)
and UCS_STATS_NODE_FREE would ignore the value that UCS_STATS_NODE_RESET writes (NULL)

Reply:
good idea, done